albumentations/augmentations/transforms.py

```diff
@@ -605,7 +606,104 @@ def get_transform_init_args_names(self):
         return ("mean", "std", "max_pixel_value")
 
 
-class Cutout(ImageOnlyTransform):
+class RandomErasing(DualTransform):
+    """ Randomly selects a rectangle region in an image and erases its pixels.
+        'Random Erasing Data Augmentation' by Zhong et al.
+        See https://arxiv.org/pdf/1708.04896.pdf
+    Args:
+         probability: The probability that the Random Erasing operation will be performed.
```

Comment:
`__init__` doesn't contain this parameter. It is named `p`

Reply:
Parameter `probability` does not make its appearance in the actual `__init__` function. Will fix the docstring.